src/components/ContactUs/MobileVersion.tsx

```diff
@@ -52,17 +81,41 @@ export default function MobileVersion() {
           />
           <textarea
             placeholder="Message"
+            name="message"
             className=" inline-block ml-2 h-[270px] resize-none outline-none flex-grow placeholder-[#7E7979] font-robotoMono text-base "
             required
           ></textarea>
         </label>
+        <ValidationError
+          prefix="Message"
+          field="message"
+          errors={state.errors}
+        />
         {/*//TODO: replace this button with the app button component */}
```

Comment:
Same as above.

Reply:
@antosan made changes as per the review with the latest commit.